homeassistant/components/heatmiser/climate.py

```diff
@@ -93,15 +89,21 @@ def hvac_mode(self) -> str:
 
         Need to be one of HVAC_MODE_*.
         """
-        return HVAC_MODE_HEAT
+        self.uh1._open()
+        self._hvac_mode = (
+            HVAC_MODE_OFF
+            if (int(self.therm.get_current_state()) == 0)
+            else HVAC_MODE_HEAT
+        )
```

Comment:
Should be removed and only handled by `update()`.

Reply:
Have amended in latest PR.